DEVELOPER.md

```diff
@@ -114,6 +114,17 @@ networks:
     assignments:              # assign static addresses to devices
       magda-core: 1
 
+# not part of magda conf, only to illustrate tunnel usage
+  - vid: 13
+    untagged: true            # don't tag this vlan - useful if the corerouter is plugged into a normal home router
+    name: uplink
+    role: ext
+    tunnel_wan_ip: 192.168.1.2/24   # put here the address of the corerouter inside the uplink network
+    tunnel_wan_gw: 192.168.1.1      # gateway of the uplink network
+    tunnel_connections: 2           # default value, number of different tunnels to create
+    tunnel_timeout: 600             # timeout in seconds after this the tunnel is destroyed and attempted to be rebuild
+    tunnel_mesh_prefix_ipv4: 10.31.142.120/29   # ip address space for both sides of the tunnel
```

Comment:
This should be explained in more detail. What is the size requirement for this ip-block? Is it tunnel_connections * 2? In addition it should also clearly state that this is a subnet derived from the main ip-block for this node.

Reply:
was technically wrong, now fixed. this is only the ip of the corerouter end of the tunnel